packages/venia-ui/lib/RootComponents/Search/search.js

```diff
@@ -1,119 +1,9 @@
-import React, { useCallback, useEffect } from 'react';
-import { object, shape, string } from 'prop-types';
-import { useQuery } from '@apollo/react-hooks';
-import { useAppContext } from '@magento/peregrine/lib/context/app';
+import React from 'react';
 
-import { mergeClasses } from '../../classify';
-import Gallery from '../../components/Gallery';
-import FilterModal from '../../components/FilterModal';
-import { fullPageLoadingIndicator } from '../../components/LoadingIndicator';
-import PRODUCT_SEARCH from '../../queries/productSearch.graphql';
-import getQueryParameterValue from '../../util/getQueryParameterValue';
-import CategoryFilters from './categoryFilters';
-import defaultClasses from './search.css';
+import SearchPage from '../../components/SearchPage';
```

Comment:
Instead of defining a new component, you may be able to just re-export `SearchPage`.

```suggestion
export { default } from '../../components/SearchPage';
```

Reply:
Done